### PR TITLE
adds forbidden lore to emagged library consoles

### PR DESCRIPTION
**Pull request by Xoxeyos**
### Intent of your Pull Request

Adds the heretic's book of forbidden knowledge to the library console.

### Why is this change good for the game?

The clockwork slab and cultists blade are also accessible in the library console, the book for forbidden knowledge should be aswell.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 

When you emag the library console, you can now get the book of forbidden knowledge.

### What should players be aware of when it comes to the changes your PR is implementing?

You can now obtain the book of forbidden knowledge when emagging a library console.

### What general grouping does this PR fall under? 
Library additions.

### Are there any aspects of the PR that you would like us not to mention on the Wiki?

Nope.

:cl:  Xoxeyos
rscadd: Emagged library console can now make heretic books.
/:cl:

### CODE REVIEW

**Comment by ChesterTheCheesy** [comment]
Why is this even a feature?

**Comment by Cartlord** [comment]
But what does the traitor gain from this?

**Comment by Xoxeyos** [comment]
It allows the traitor to commit acts of subterfuge, people may think there's infact a blood cult, or a clock cult on the station, and I figured I'd add in the ability to generate these books as well. And with dynamic on the way to being completed, should a cult member gain access to an emag, they can now create new items.
